src/main/java/net/mcreator/ui/modgui/PlantGUI.java

```diff
@@ -1,801 +1,844 @@
-		plant.hardness = (double) hardness.getValue();
-		plant.resistance = (double) resistance.getValue();
-		plant.luminance = (int) luminance.getValue();
-		plant.unbreakable = unbreakable.isSelected();
-		plant.forceTicking = forceTicking.isSelected();
-		plant.hasTileEntity = hasTileEntity.isSelected();
-		plant.soundOnStep = new StepSound(mcreator.getWorkspace(), soundOnStep.getSelectedItem());
-		plant.useLootTableForDrops = useLootTableForDrops.isSelected();
-		plant.customDrop = customDrop.getBlock();
-		plant.dropAmount = (int) dropAmount.getValue();
-		plant.frequencyOnChunks = (int) frequencyOnChunks.getValue();
-		plant.onBlockAdded = onBlockAdded.getSelectedProcedure();
-		plant.onNeighbourBlockChanges = onNeighbourBlockChanges.getSelectedProcedure();
-		plant.onTickUpdate = onTickUpdate.getSelectedProcedure();
-		plant.onRandomUpdateEvent = onRandomUpdateEvent.getSelectedProcedure();
-		plant.onDestroyedByPlayer = onDestroyedByPlayer.getSelectedProcedure();
-		plant.onDestroyedByExplosion = onDestroyedByExplosion.getSelectedProcedure();
-		plant.onStartToDestroy = onStartToDestroy.getSelectedProcedure();
-		plant.onEntityCollides = onEntityCollides.getSelectedProcedure();
-		plant.onBlockPlacedBy = onBlockPlacedBy.getSelectedProcedure();
-		plant.onRightClicked = onRightClicked.getSelectedProcedure();
-		plant.spawnWorldTypes = spawnWorldTypes.getListElements();
-		plant.restrictionBiomes = restrictionBiomes.getListElements();
-		plant.isReplaceable = isReplaceable.isSelected();
-		plant.colorOnMap = (String) colorOnMap.getSelectedItem();
-		plant.offsetType = (String) offsetType.getSelectedItem();
-		plant.aiPathNodeType = (String) aiPathNodeType.getSelectedItem();
-		plant.creativePickItem = creativePickItem.getBlock();
-		plant.flammability = (int) flammability.getValue();
-		plant.fireSpreadSpeed = (int) fireSpreadSpeed.getValue();
-		plant.specialInfo = StringUtils.splitCommaSeparatedStringListWithEscapes(specialInfo.getText());
-		plant.generateCondition = generateCondition.getSelectedProcedure();
-		plant.emissiveRendering = emissiveRendering.isSelected();
-
-		plant.customBoundingBox = customBoundingBox.isSelected();
-		plant.disableOffset = disableOffset.isSelected();
-		plant.boundingBoxes = boundingBoxList.getBoundingBoxes();
-
-		Model model = Objects.requireNonNull(renderType.getSelectedItem());
-		plant.renderType = 12;
-		if (model.getType() == Model.Type.JSON)
-			plant.renderType = 2;
-		else if (model.getType() == Model.Type.OBJ)
-			plant.renderType = 3;
-		else if (model.equals(cross))
-			plant.renderType = "No tint".equals(tintType.getSelectedItem()) ? 12 : 120;
-		else if (model.equals(crop))
-			plant.renderType = 13;
-		plant.customModelName = model.getReadableName();
-
-		return plant;
-	}
-
-	@Override public @Nullable URI getContextURL() throws URISyntaxException {
-		return new URI(MCreatorApplication.SERVER_DOMAIN + "/wiki/how-make-plant");
-	}
-
-}
+/*
```

Comment:
Please consider reverting this file and applying changes again. Currently, the patch of this PR overrides the file which will remove file history.

Reply:
The problem is that I've already done that. I took the file from GitHub, applied my changes and it does that.

Reply:
Could be line endings: https://stackoverflow.com/questions/19593909/git-diff-sees-whole-file-as-changed-when-its-not, https://stackoverflow.com/questions/30752959/git-showing-entire-file-is-modified-instead-of-showing-modified-small-part-of-co

This may help too: https://stackoverflow.com/questions/5787937/git-status-shows-files-as-changed-even-though-contents-are-the-same

I can't accept PR until proper file history is kept.